lib/deviceinfo.lib

```diff
@@ -132,7 +132,7 @@ getDeviceInfo() {
   [[ -n "${DEVPROP[ro.modversion]}" ]] && echo "* Mod: ${DEVPROP[ro.modversion]}" >> "$devicedoc" # ro.modversion=10-20121228-NIGHTLY-milestone2
   [[ -n "${DEVPROP[ro.build.id]}" ]] && echo "* BuildID: ${DEVPROP[ro.build.id]}" >> "$devicedoc" # JZO54K
   echo "* Fingerprint: ${DEVPROP[ro.build.fingerprint]}" >> "$devicedoc" # MOTO/A953_O2DE/milestone2:2.3.6/4.5.1-134_DFP-132/1317968148:user/release-keys
-  echo "* Kernel: $(adb ${ADBOPTS} shell uname -r)" >> "$devicedoc"
+  [[ "$(adb ${ADBOPTS} shell whence uname)" ]] && echo "* Kernel: $(adb ${ADBOPTS} shell uname -r)" || echo "* Kernel: $(adb ${ADBOPTS} shell busybox uname -r)" >> "$devicedoc"
```

Comment:
just wondering: is this really necessary with Busybox installed *correctly*? On install, Busybox usually creates the appropriate symlinks. Also, are you sure `whence` is available on all ROMs? Wouldn't be nice to break things where a correct install of Busybox would be the better approach :wink: